package.json

```diff
@@ -4,37 +4,50 @@
   "description": "Koa Pagination",
   "license": "MIT",
   "author": "Seegno",
-  "main": "index.js",
+  "main": "dist/src",
   "repository": "git@github.com:seegno/koa-pagination",
   "scripts": {
     "changelog": "github_changelog_generator --no-issues --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,4ba' -e 'P;D' CHANGELOG.md",
-    "test": "NODE_ENV=test mocha --recursive",
-    "version": "npm run changelog --future-release=$npm_package_version && git add -A CHANGELOG.md"
+    "lint": "eslint src test && jscs src test",
+    "prepublish": "npm run transpile",
+    "test": "NODE_ENV=test mocha $npm_package_options_mocha",
+    "transpile": "rm -rf dist/* && babel src --out-dir dist/src",
+    "version": "npm run changelog --future-release=$npm_package_version && npm run transpile && git add -A CHANGELOG.md dist"
   },
   "dependencies": {
     "debug": "2.2.0",
     "http-content-range-format": "1.0.0",
-    "is-safe-integer": "^1.0.1",
-    "lodash": "4.12.0",
+    "is-safe-integer": "1.0.1",
```

Comment:
Not needed as node 4 (which this package requires) already ships `Number.isSafeInteger`.